_posts/2022-05-16-an-opinionated-guide-to-consolidating-our-data.md

```diff
@@ -0,0 +1,234 @@
+---
+layout: post
+title:  "An Opinionated Guide to Consolidating our Data"
+author: Abhi Vaidyanatha
+excerpt_separator: <!--more-->
```

Comment:
Add 

canonical_url: 'https://abhi-vaidyanatha.medium.com/an-opinionated-guide-to-consolidating-your-data-b09386b2b9b5'